README.md

```diff
@@ -66,7 +65,7 @@ Also check out the sister project, [awesome-dynamic-analysis](https://github.com
 * [splint](https://github.com/ravenexp/splint) - Annotation-assisted static program checker
 * [vera++](https://bitbucket.org/verateam/vera/wiki/Introduction) - Vera++ is a programmable tool for verification, analysis and transformation of C++ source code.
 
-## C# #
+## C
```

Comment:
The hashes at the end actually were on purpose. Otherwise it would be `C` instead of `C#`. 😉